erpnext/patches/v13_0/item_reposting_for_incorrect_sl_and_gl.py

```diff
@@ -12,6 +12,7 @@ def execute():
 			("stock", "purchase_receipt_item"),
 			("stock", "delivery_note_item"),
 			("stock", "packed_item"),
+			("stock", "stock_ledger_entry"),
```

Comment:
This is already done in a separate patch. It's no wise to reload it here.

Schema for SLE actually changed between v12 and v13 and hence the patch that migrates schema should only touch it.

Reply:
The patches were still breaking during migration.. do we need to reorder the patch then?

Reply:
the patch already runs before this one it's called "update_is_cancelled_field"

Reply:
https://github.com/frappe/erpnext/blob/964b911b32d71e3069ad0f942b447cd25cb4da1c/erpnext/patches/v12_0/update_is_cancelled_field.py#L12

Reply:
Ahh, this is strange. Nvm, I will remove it. Thanks!